src/Giraffe/Giraffe.fsproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <!-- General -->
     <AssemblyName>Giraffe</AssemblyName>
+    <AssemblyVersion>7.0.2</AssemblyVersion>
```

Comment:
Another potential fix would be to define this property at the `dotnet pack` command on CI here https://github.com/giraffe-fsharp/Giraffe/blob/master/.github/workflows/publish.yml#L69